dask/local.py

```diff
@@ -61,12 +61,16 @@
 Examples
 --------
 
->>> import pprint  # doctest: +SKIP
->>> dsk = {'x': 1, 'y': 2, 'z': (inc, 'x'), 'w': (add, 'z', 'y')}  # doctest: +SKIP
->>> pprint.pprint(start_state_from_dask(dsk))  # doctest: +SKIP
+>>> import pprint
+>>> dsk = {'x': 1, 'y': 2, 'z': (inc, 'x'), 'w': (add, 'z', 'y')}
+>>> pprint.pprint(start_state_from_dask(dsk))
 {'cache': {'x': 1, 'y': 2},
  'dependencies': {'w': {'z', 'y'}, 'x': set(), 'y': set(), 'z': {'x'}},
- 'dependents': {'w': set(), 'x': {'z'}, 'y': {'w'}, 'z': {'w'}},
+ 'dependents': defaultdict(None,
+                           {'w': set(),
+                            'x': {'z'},
+                            'y': {'w'},
+                            'z': {'w'}}),
```

Comment:
I had to spread these out across multiple lines to match `pprint`s formatting. Tried the following thinking it would ignore white spaces to no avail. If there's better way, let me know.
https://docs.python.org/3/library/doctest.html#doctest.NORMALIZE_WHITESPACE

```python
>>> pprint.pprint(start_state_from_dask(dsk))  # doctest: +NORMALIZE_WHITESPACE
```

Reply:
Done. Reverted to the original skipping behavior.